robocop/checkers/comments.py

```diff
@@ -150,19 +170,12 @@ def check_invalid_comments(self, name, node):
 
     def check_comment_content(self, token, content):
         content = content.lower()
-        if "todo" in content:
-            self.report(
-                "todo-in-comment",
-                todo_or_fixme="TODO",
-                lineno=token.lineno,
-                col=token.col_offset + 1 + content.find("todo"),
-            )
-        if "fixme" in content:
+        for marker in [term for term in self.todos if term.lower() in content]:
```

Comment:
To avoid doing term.lower() for every comment, we can do it when defining todo list:
```
self._todos = [term.lower() for term in self.param("todo-in-comment", "todos").split(",")]
```
and then
```
for marker in [term for term in self.todos if term in content]:
```

Reply:
Good point.